council/llm/llm_base.py

```diff
@@ -1,18 +1,40 @@
 import abc
 import logging
 
-from typing import List, Any
-from .llm_message import LLMMessage
+from typing import List, Any, Optional, Sequence
+from .llm_message import LLMMessage, LLMessageTokenCounterBase
+from ..runners.budget import Consumption
 
 logger = logging.getLogger(__name__)
 
 
+class LLMResult:
+    def __init__(self, choices: List[str], consumptions: Optional[List[Consumption]] = None):
+        self._choices = choices
+        self._consumptions = consumptions if consumptions is not None else []
+
+    @property
+    def first_choice(self) -> str:
+        return self._choices[0]
```

Comment:
Can `_choices` be empty? If yes, should we return `Optional[str]`?

Reply:
The call is returning an LLResult when the request succeed. So we are sure to have at list a choice. 
Next, the impact on the client code could be a bit cumbersome to test for None
The worst case is to have a choice but empty.